client/components/MapProducts.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Card from 'react-bootstrap/Card';
+import Col from 'react-bootstrap/Col';
+
+export default function MapProducts(props) {
+  return props.products.map((product) => {
```

Comment:
Minor thing here, but I would probably destructure products directly in the function params above